src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java

```diff
@@ -301,7 +302,17 @@ private void parseAdminPlotCommand(String[] split) throws TownyException {
 		}
 
 		if (split[0].equalsIgnoreCase("meta")) {
-			handlePlotMetaCommand(player, split);
+
+			if (!townyUniverse.getPermissionSource().testPermission(player, PermissionNodes.TOWNY_COMMAND_TOWNYADMIN_PLOT_CLAIM.getNode()))

```

Comment:
Still have a permission node check for a node that doesn't match/doesn't need to be done here.